src/vs/code/electron-main/app.ts

```diff
@@ -487,9 +493,6 @@ export class CodeApplication {
 			}
 		}
 
-		// Install Menu
-		appInstantiationService.createInstance(CodeMenu);
```

Comment:
@sbatten is the functionality from `vs/code/electron-main/menus.ts` moved over to `menubar.ts`? then I think it would be safe to remove `menu.ts`?

However I see that some things are not working yet, like the Accessibility help window, so maybe that is still work in progress?